libmscore/instrument.cpp

```diff
@@ -26,8 +26,8 @@
 
 namespace Ms {
 
-const char* Channel::DEFAULT_NAME = QT_TRANSLATE_NOOP("channel", "Normal");
-const char* Channel::HARMONY_NAME = QT_TRANSLATE_NOOP("channel", "Chord symbols");
+const char* Channel::DEFAULT_NAME = QT_TRANSLATE_NOOP("InstrumentsXML", "Normal");
+const char* Channel::HARMONY_NAME = QT_TRANSLATE_NOOP("InstrumentsXML", "Chord symbols");
```

Comment:
Using the same context as the rest of the instruments, long- and short names, channels, etc..
Simplifies the code that picks up those translations quite a but, but means that the current translations won't catch it (as they  have a different context), so needs to get merged and translated to be able to see whether it then shows the translated names in the UI.

Reply:
Wonder whether to use `QT_TRANSLATE_NOOP("InstrumentsXML", "Normal"`, "Channel")` to give some hint to the translator?